webapp/src/main/resources/public/js/ict/chrome-ict-popup.js

```diff
@@ -25,6 +25,15 @@ const IctPopup = function (props) {
                 <FormControl type="text" value={props.mojitoBaseUrl} onChange={(e) => props.onMojitoBaseUrlChanged(e.target.value)} />
               </Col>
             </FormGroup>
+
+            <FormGroup controlId="formHorizontalMojitoUrl">
```

Comment:
oh, you made that configurable?  I think it makes things a little bit more complicated and we don't really need it for now. I'd suggest removing it